### ISSUE TITLE
Improve app config

### ISSUE THREAD

**Issue by chrisvdg** [chrisvdg/scrumtime#7]
The app config now consist of 2 top level parts: `messengers` for the chat platform api configs and `schedules` for the messages.
First idea was to rename `schedules` to a more logical `messages` but this can easy be mistaken with `messengers`.

Todo:
* Update `App` struct and usage
* Update documentation (readme)

**Comment by ilmanzo** [comment]
this would lead to 'messengers' and 'messages' , I think it's a bit confusing ... How about 'senders' and 'messages' ? 

```
// App represents the app's configuration
type App struct {
	Messengers map[string]*Messenger `yaml:"messengers"`
	Messages   map[string]*Schedule  `yaml:"messages"`
}
```

**Comment by chrisvdg** [comment]
Actually was thinking we need more refactoring especially looking at #6  if we were to allow to start without any schedule messages and add them interacting with webhook, `getupdates`, ... api's of the platforms.

Thinking of making the config yaml file to look like:
```yaml
bots:
  example_telegram_bot:
    platform: telegram
    api_key: xxx

messages:
  example_message:
    body: "Hello world"
    messengers:
      - bot: example_telegram_bot
        chat_ids: 
          - "-123456789"
          - "-100987654"
```

**Comment by chrisvdg** [comment]
closed by 66d7d46
